### PR TITLE
8159454: [TEST_BUG] javax/swing/ToolTipManager/7123767/bug7123767.java: number of checked graphics configurations should be limited

### PR DESCRIPTION
**Pull request by ktakakuri**
Hi all,

This is a clean backport of JDK-8159454: [TEST_BUG] javax/swing/ToolTipManager/7123767/bug7123767.java: number of checked graphics configurations should be limited.
I have confirmed that bug7123767.java passes by backporting this fix.

Thank you.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8159454](https://bugs.openjdk.org/browse/JDK-8159454) needs maintainer approval

### Issue
 * [JDK-8159454](https://bugs.openjdk.org/browse/JDK-8159454): [TEST_BUG] javax/swing/ToolTipManager/7123767/bug7123767.java: number of checked graphics configurations should be limited (**Bug** - P4 - Approved)

### Reviewers
 * [Andrew John Hughes](https://openjdk.org/census#andrew) (@gnu-andrew - **Reviewer**) ⚠️ Review applies to [38bde236](https://git.openjdk.org/jdk8u-dev/pull/405/files/38bde236171dbddf5370bd9ba03a6b57ba407c64)
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**) ⚠️ Review applies to [38bde236](https://git.openjdk.org/jdk8u-dev/pull/405/files/38bde236171dbddf5370bd9ba03a6b57ba407c64)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk8u-dev.git pull/405/head:pull/405` \
`$ git checkout pull/405`

Update a local copy of the PR: \
`$ git checkout pull/405` \
`$ git pull https://git.openjdk.org/jdk8u-dev.git pull/405/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 405`

View PR using the GUI difftool: \
`$ git pr show -t 405`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk8u-dev/pull/405.diff">https://git.openjdk.org/jdk8u-dev/pull/405.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk8u-dev/pull/405#issuecomment-1855751471)

### CODE REVIEW

File: jdk/test/javax/swing/ToolTipManager/7123767/bug7123767.java [hunk 1]
```diff
@@ -22,10 +22,25 @@
...
+*           then some 5 random configurations will be checked.
+*           Please Use -Dseed=X to set the random generator seed
+*           (if necessary).
+*
+* @author   Vladislav Karnaukhov
+*
+* @key      headful
+* @key      randomness
+*
+* @modules  java.desktop/sun.awt
```
Comment (mrserb) [inline jdk/test/javax/swing/ToolTipManager/7123767/bug7123767.java:39]: "@modules" tag is not needed for jdk8

**Comment by mrserb** [comment]
keep open

**Review by gnu-andrew — APPROVED** [review]
Patch looks clean, bar the header and the indentation changes around `robot.waitIdle()`. I confirmed all but the header are the same as 11u once this patch is applied.

As to `@modules`, we can either drop it here or do it as part of a cleanup in bulk as for #441

**Review by mrserb — APPROVED** [review]

**Comment by ktakakuri** [comment]
/approval request This fix applis cleanly to 8u from jdk9,
There is no risk, because the logic around the changes and the cause of the problem is the same as in jdk9,
and I have confirmed that bug7123767.java passes by backporting the fix.

**Comment by gnu-andrew** [comment]
I guess we're removing `@modules` elsewhere then.
/approve yes

**Comment by ktakakuri** [comment]
After requesting the fix request I made an additional commit, which is a fix for a syntax error.

**Pull request closed without merging**